sympy/combinatorics/tests/test_fp_groups.py

```diff
@@ -830,21 +830,28 @@ def test_subgroup_presentations():
+        " b_1**-2*c_3**-1*b_1*c_3*b_1**2*c_3*b_1**-2*c_3**-1*b_1*c_3*b_1**2*c_3,"
+        " b_1*c_3**-1*b_1**-2*c_3**-1*b_1**-1*c_3*b_1**2*c_3**-1*b_1**-2*c_3**-1*b_1**-1*c_3*b_1,"
+        " b_1**-1*c_3**-1*b_1*c_3*b_1**2*c_3*b_1**-2*c_3**-1*b_1*c_3**-1*b_1*c_3*b_1**2*c_3*b_1**-2*c_3**-1*b_1**2*c_3,"
+        " b_1**-1*c_3*b_1**2*c_3*b_1**-2*c_3**-1*b_1**-1*c_3**-1*b_1*c_3*b_1**2*c_3*b_1**2*c_3**-1*b_1*c_3*b_1**2*c_3*b_1**-2,"
+        " b_1*c_3**-1*b_1*c_3*b_1**2*c_3*b_1**-2*c_3*b_1**2*c_3**-1*b_1**-2*c_3**-1*b_1**2*c_3*b_1**2*c_3*b_1**-2*c_3**-1,"
+        " c_3*b_1**2*c_3**-1*b_1**-2*c_3**-1*b_1**-1*c_3**-1*b_1**-2*c_3**-1*b_1**-1*c_3**-1*b_1**2*c_3**-1*b_1**-2*c_3**-1*b_1**-1"
+        "*c_3**-1*b_1**-2*c_3**-1*b_1**-1*c_3, b_1**-1*c_3*b_1**2*c_3*b_1**-2*c_3**-1*b_1**2*c_3*b_1**2*c_3*b_1**-2*c_3**-1*b_1**2"
+        "*c_3*b_1**2*c_3*b_1**-2*c_3**-1*b_1**-2, b_1**-1*c_3**-1*b_1*c_3*b_1**2*c_3*b_1**-2*c_3**-1*b_1**-1*c_3**-1*b_1*c_3*b_1**2"
+        "*c_3*b_1**-2*c_3**-1*b_1**-1*c_3**-1*b_1*c_3*b_1**2*c_3*b_1**-2*c_3**-1, b_1**-1*c_3**-1*b_1*c_3*b_1**2*c_3*b_1*c_3*b_1**2*c_3"
+        "*b_1**-2*c_3**-1*b_1**-1*c_3*b_1**2*c_3**-1*b_1**-2*c_3**-1*b_1**-1*c_3**-1*b_1**-2*c_3**-1*b_1**-1*c_3,"
+        " c_3*b_1**2*c_3**-1*b_1**-2*c_3**-1*b_1**-1*c_3**-1*b_1**-2*c_3**-1*b_1**-1*c_3*b_1*c_3*b_1**2"
+        "*c_3**-1*b_1**-2*c_3**-1*b_1**-1*c_3**-1*b_1**-2*c_3**-1*b_1**-1*c_3*b_1, b_1*c_3*b_1**2*c_3**-1*b_1**-2*c_3**-1*b_1**-1*c_3**-1"
+        "*b_1**-2*c_3**-1*b_1**-1*c_3*b_1**2*c_3*b_1**2*c_3**-1*b_1**-2*c_3**-1*b_1**-1*c_3**-1*b_1**-2*c_3**-1*b_1**-1*c_3*b_1,"
+        " b_1**2*c_3**-1*b_1*c_3*b_1**2*c_3**-1*b_1**2*c_3**-1*b_1*c_3*b_1**2*c_3**-1*b_1**2*c_3**-1*b_1*c_3*b_1**2"
+        "*c_3**-1*b_1**2*c_3**-1*b_1*c_3*b_1**2*c_3**-1*b_1**-2*c_3*b_1**-2*c_3**-1*b_1**-1*c_3, b_1**-1*c_3**-1*b_1*c_3*b_1**2*c_3**-1"
+        "*b_1**2*c_3**-1*b_1*c_3*b_1**2*c_3**-1*b_1**2*c_3**-1*b_1*c_3*b_1**2*c_3**-1*b_1**2*c_3**-1*b_1*c_3*b_1**2*c_3**-1*b_1**2"
+        "*c_3**-1*b_1*c_3*b_1**2*c_3**-1*b_1**2*c_3**-1*b_1*c_3*b_1**2*c_3**-1*b_1**2*c_3**-1*b_1*c_3*b_1**2*c_3**-1*b_1**2*c_3**-1*b_1"
+        "*c_3*b_1**2*c_3**-1*b_1**2*c_3**-1*b_1*c_3*b_1**2*c_3**-1*b_1**2*c_3**-1*b_1*c_3*b_1**2*c_3**-1*b_1**2*c_3**-1*b_1*c_3*b_1**2"
+        "*c_3**-1*b_1**-2, b_1**-1*c_3**-1*b_1*c_3*b_1**2*c_3**-1*b_1**2*c_3**-1*b_1*c_3*b_1**2*c_3**-1*b_1**2*c_3**-1*b_1*c_3*b_1**2"
+        "*c_3**-1*b_1**2*c_3*b_1**2*c_3**-1*b_1**-2*c_3**-1*b_1**-1*c_3*b_1**2*c_3**-1*b_1*c_3*b_1**2*c_3**-1*b_1**2*c_3**-1*b_1*c_3"
+        "*b_1**2*c_3**-1*b_1**2*c_3**-1*b_1*c_3*b_1**2*c_3**-1*b_1**2*c_3*b_1**2*c_3**-1*b_1**-2*c_3**-1*b_1**-1*c_3*b_1**2*c_3**-1*b_1"
+        "*c_3*b_1**2*c_3**-1*b_1**2*c_3**-1*b_1*c_3*b_1**2*c_3**-1*b_1**2*c_3**-1*b_1*c_3*b_1**2*c_3**-1*b_1**2*c_3*b_1**2*c_3**-1*b_1**-2"
+        "*c_3**-1*b_1**-1*c_3*b_1**-2))")
     assert str(reidemeister_presentation(f, H)) == k
```

Comment:
Maybe we could have a more readable test like this, for example:
```
gens, rels = reidemeister_presentation(f, H)
assert gens == ...
assert len(rels) == ...
```
The list of relators is not very informative.